### ISSUE TITLE
Do you use a val/test split?

### ISSUE THREAD

**Issue by slerman12** [hysts/pytorch_image_classification#18]
Hi, in your reported results, are these (1) best performing test accuracies without a validation set, (2) final test accuracies on the last epoch of training, or (3) test accuracies on the model checkpoint with the best performing validation accuracy?

**Comment by hysts** [comment]
Hi, @slerman12 

All of the results are (2) final test accuracies on the last epoch of training.
